UnityGLTF/Assets/UnityGLTF/Runtime/Shaders/VertexColor.cginc

```diff
@@ -0,0 +1,70 @@
+#ifndef VERTEX_COLOR_INCLUDED
+#define VERTEX_COLOR_INCLUDED
+
+#include "UnityCG.cginc"
+#include "UnityStandardInput.cginc"
+#include "UnityStandardCoreForward.cginc"
+
+struct WrappedVertexOutputForwardBase {
+    VertexOutputForwardBase innerValue;
+    fixed4 color : COLOR;
+};
+
+WrappedVertexOutputForwardBase vert_vcol (appdata_full v)
+{
+    // Save the color information
+    WrappedVertexOutputForwardBase o;
+    o.color = v.color;
+
+    // Forward this call to the default vertBase
+    VertexInput vi;
+    vi.vertex = v.vertex;
+    vi.normal = v.normal;
+    vi.uv0 = v.texcoord;
+    vi.uv1 = v.texcoord1;
+#if defined(DYNAMICLIGHTMAP_ON) || defined(UNITY_PASS_META)
+    vi.uv2 = v.tetcoord2;
+#endif
+#ifdef _TANGENT_TO_WORLD
+    vi.tangent = v.tangent;
+#endif
+    o.innerValue = vertBase(vi);
+
+    return o;
+}
+
+fixed4 frag_vcol(WrappedVertexOutputForwardBase wvofb) : SV_Target
+{
+    // Put the original output from vertBase in a variable called 'i' for the macros
+    // (e.g. FRAGMENT_SETUP) that assume that naming convention.
+    VertexOutputForwardBase i = wvofb.innerValue;
+
+    // The following section is copied from the fragBase implementation, but with
+    // an inserted section to modify the diffColor after it is calculated but before

```

Comment:
nit: bonus points if you link to the applicable file in the official Unity gitbucket repo.